common/changes/@energyweb/semantic-release-config/bug-ICL-335_skipped_release_notes_2022-08-02-11-31.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@energyweb/semantic-release-config",
+      "comment": "Add documentation and performance commits to release notes",
```

Comment:
```suggestion
      "comment": "Add documentation and performance commits to release notes. Also add perf and remove ci and build from release rules",
```

Reply:
Double quotes were changed by single quotes according to our prettier config. We can do it in separate PR though

Reply:
Don't know why git shows changes in eslint-config package. This PR actually doesn't change anything there